[#2691] Add templates by portals (#3018)
GeotrekCE/Geotrek-admin: docs/advanced-configuration.rst

    {% block picto_attr %}style="background-color: {{ object.practice.color }};"{% endblock picto_attr %}

CSS can be overriden like html templates: copy them to ``var/media/templates/trekking/`` or ``var/media/templates/tourism/`` folder
``/opt/geotrek-admin/var/conf/extra_templates/trekking/trek_public_pdf.css`` for example.

You can also create a template for each portal.

Add a folder ``portal_{id_portal}`` (you can find ids of your portal in ``/admin/common/targetportal/{id_portal}``) in

> ``/opt/geotrek-admin/var/conf/extra_templates/<appname>``, as the first template add at the top of your file:

Reviewer: ```suggestion
``/opt/geotrek-admin/var/conf/extra_templates/<appname>``, as the first template, and add at the top of your file:
```

::

    {% extends "trekking/trek_public_pdf.html" %}

> Your template for your specific portal will use the modification you made on your overriden template in ``/opt/geotrek-admin/var/conf/extra_templates/<appname>``

Reviewer: ```suggestion
The template for a specific portal will use the modification made on the overriden template in ``/opt/geotrek-admin/var/conf/extra_templates/<appname>``
```
